### DISCUSSION TITLE
Check out multiple QTY of Consumables/Accessories

### DISCUSSION THREAD

**Discussion by Kitasan-Black-Home** [snipe/snipe-it#15041] [Ideas]
Good morning,

I saw this mentioned in 2021 but have not seen any mention of it since.  I believe this would be extremely useful when it comes to things such as spools of ethernet cable or Fiber.   When you have to manually determine the length of a cable, run it and tip it then it would be awesome to be able to use the Checkout feature to subtract how much was used from the overall quantity instead of manually editing it each time.   I know you can do that with Components so hopefully it would not be too difficult to add that option to other categories as well.

**Comment by TShiremanLCOG** [comment]
Greetings - Checking out multiple Consumables is possible in at least version 7.0.8. You can do it with Accessories with a terrible workaround but I wouldn't recommend it.
